apps/mac/FluentUIReactNative-macOSUITests/FluentUIReactNative_macOSUITests.m

```diff
@@ -0,0 +1,48 @@
+//
+//  FluentUIReactNative_macOSUITests.m
+//  FluentUIReactNative-macOSUITests
+//
+//  Created by Chris Hogan on 3/24/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface FluentUIReactNative_macOSUITests : XCTestCase
```

Comment:
Can we add tests in a separate change? Or at least not add dummy tests?

But also they probably should be swift?

Reply:
The test templates I have now make adding them that much easier when I get to that stage of this project. Don't see a benefit to deleting something I'll need later creating more work for myself?

Reply:
We're not ever going to add our UITests in a monolithic FluentUIReactNative_macOSUITests.m I hope - we would instead add more specific tests in better named files. And ideally we would do it in swift.

Plus the templates get autocreated if you create a target, so this is trivially recreatable. Figured just removing now would be easier than trying to come up with the name of one suite of tests we want so removing it seemed like the best way to scope down this change without adding known incorrect code.

Reply:
I'll recreate this all in swift. I won't add tests.